index.js

```diff
@@ -98,6 +112,37 @@ function Collection(manager, name){
 
 Collection.prototype.__proto__ = MonkCollection.prototype;
 
+Collection.prototype.ensureEventEmitterExists = function() {
```

Comment:
Is this somehting we added for compatibility reasons? If not, it feels like it's use could be more of a lazyload getter. That returns the emitter.